app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
`Elf` class implements not all abstract methods from `Player`, thus being abstract as well. Inherit the `Elf` class from `ABC` too.